packages/@lwc/babel-plugin-component/src/decorators/index.ts

```diff
@@ -105,28 +113,30 @@ function validate(decorators) {
 }
 
 /** Remove import specifiers. It also removes the import statement if the specifier list becomes empty */
-function removeImportedDecoratorSpecifiers(engineImportSpecifiers) {
+function removeImportedDecoratorSpecifiers(
+    engineImportSpecifiers: { name: any; path: NodePath<Node> }[]
+) {
     engineImportSpecifiers
         .filter(({ name }) => isLwcDecoratorName(name))
         .forEach(({ path }) => {
-            const importStatement = path.parentPath;
+            const importStatement = path.parentPath as NodePath<types.ImportDeclaration>;
             path.remove();
             if (importStatement.get('specifiers').length === 0) {
                 importStatement.remove();
             }
         });
 }
 
-function generateInvalidDecoratorError(path) {
+function generateInvalidDecoratorError(path: NodePath<types.Decorator>) {
     const expressionPath = path.get('expression');
     const { node } = path;
     const { expression } = node;
 
     let name;
     if (expressionPath.isIdentifier()) {
-        name = expression.name;
+        name = (expression as any).name;
     } else if (expressionPath.isCallExpression()) {
-        name = expression.callee.name;
+        name = (expression as any).callee.name;
```

Comment:
This looks like a bug. `callee` could be something other than an `Identifier`, looks like. It's more obvious with the types here. No need to fix it - you explicitly called out no functional changes.

Reply:
Replaced with this FWIW:

```ts
name = ((expression as types.CallExpression).callee as types.V8IntrinsicIdentifier).name;
```